packages/o11ylogsdb/bench/synthetic-corpora.ts

```
@@ -0,0 +1,365 @@
+ */
+
+import type { AnyValue, LogRecord } from "../dist/index.js";
+
+// ── Random helpers ───────────────────────────────────────────────────
+
+let _seed = 42;
+function setSeed(s: number) {
+  _seed = s;
+}
+function rand(): number {
+  let t = (_seed += 0x6d2b79f5);
```

Biome (2.4.13)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)